tests/conftest.py

```diff
@@ -140,7 +139,7 @@ def mock_session(
 @pytest.fixture
 def mock_ote_response(
     requests_mock: "Mocker", server_url: str
-) -> "Callable[[Union[HTTPMethod, str], str, int, Optional[Union[Dict[str, Any], str]], Optional[dict], Optional[bytes], Optional[Union[dict, str]], Optional[str], Optional[OTEClient]], None]":  # pylint: disable=line-too-long
+) -> "Callable[[Union[HTTPMethod, str], str, int, Optional[Union[Dict[str, Any], str]], Optional[dict], Optional[bytes], Optional[Union[dict, str]], Optional[str], Optional[OTEClient]], None]":  # noqa: E501
```

Comment:
what is noqa? is that the same of pylint disable?

Reply:
Indeed. It is the more generally used code for disabling some linting (or dev tool) rule. It means "no quality assurance" (apparently, after googling). Rule E501 is the codename for "line-too-long".

I'm generally against using codenames for rules instead of the "names", but I don't think that's possible with ruff 😞